Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -377,44 +377,131 @@ class LayoutUtility
 {
   public static GetHeight(element: Element)
   {
-    return LayoutUtility.GetElementDimension(element, "height");
+    return LayoutUtility.GetElementDimensionWithElementShow(element, "height");
+  }
+
+  public static GetInnerHeight(element: Element)
+  {
+    return LayoutUtility.GetElementDimensionWithElementShow(element, "height", "padding");
   }
 
   public static GetWidth(element: Element)
   {
-    return LayoutUtility.GetElementDimension(element, "width");
+    return LayoutUtility.GetElementDimensionWithElementShow(element, "width");
+  }
+
+  public static GetInnerWidth(element: Element)
+  {
+    return LayoutUtility.GetElementDimensionWithElementShow(element, "width", "padding");
   }
 
-  private static GetElementDimension(element: Element, dimension: "width" | "height")
+  public static GetOuterHeight(element: HTMLElement)
+  {
+    return this.GetElementDimensionWithElementShow(element, "height", "border");
+  }
+
+  public static GetOuterWidth(element: HTMLElement)
+  {
+    return this.GetElementDimensionWithElementShow(element, "width", "border");
+  }
+
+  private static GetElementDimensionWithElementShow(element: Element, dimension: "width" | "height", extra?: "padding" | "border")
```

Comment:
Please rename for better understanding of the name. `...WithElementShow` doesn't really help me understand what the method does.